src/status_im/chat/events/input.cljs

```diff
@@ -439,42 +439,58 @@
                                       [::send-command (assoc-in command-message
                                                                 [:command :preview] returned)])})))))
 
+(defn command-complete?
+  [chat-command]
+  (= :complete (input-model/command-completion chat-command)))
+
+(defn command-complete-fx
+  "command is complete, clear sequential arguments and proceed with command processing"
+  [db chat-command message-id current-time]
+  (-> db
+      clear-seq-arguments
+      (model/set-chat-ui-props {:sending-in-progress? true})
+      (proceed-command chat-command message-id current-time)))
+
+(defn command-not-complete-fx
+  "command is not complete, just add space after command if necessary"
+  [db input-text]
+  {:db (cond-> db
+         (not (input-model/text-ends-with-space? input-text))
+         (set-chat-input-text constants/spacing-char :append? true))})
+
+(defn no-command-fx
```

Comment:
Not a big deal, but instead of `no-command-fx`, I would rather name this function `text-message-fx` or `plain-message-fx`.

Reply:
ok will fix, but what should we do with this PR ? It works on simulator but on device it has no effect apparently. I guess I could rename it and just consider it a refactor to make this part more readable and take another approach in a new PR ?

Reply:
Sure, it's super minor thing and I still approve the PR as it is, eq without renaming

Reply:
How about we test + merge and then do other work in new PR?